### PR TITLE
sorted table

### PR DESCRIPTION
**Pull request by AlexiLynn**
Hello team, this adds a roommate table that sorts by points! It currently doesn't return householdId correctly and I am not sure why. Please let me know if you have any ideas. If not we can always fix it when we meet this weekend. Thanks!

### CODE REVIEW

File: packages/frontend/src/pages/LogIn.js [hunk 1]
```diff
@@ -30,6 +30,7 @@ const LogIn = () => {
         //storing token in localStorage
         localStorage.setItem("token", responseData.token);
         localStorage.setItem("userId", responseData.userId);
+        localStorage.setItem("householdId", responseData.householdId);
```
Comment (nicoleruedger) [inline packages/frontend/src/pages/LogIn.js:33]: Does responseData hold householdId? What is the expected response and what does it hold?

File: packages/frontend/src/components/Layout.js [hunk 2]
```diff
@@ -1,10 +1,12 @@
 import React from "react";
 import styles from "./LayoutStyle.module.css";
-import HardcodedTable from "./Table.js";
+import UsersTable from "./Table.js";
 import HardcodedTodoList from "./ToDo";
 import Profile from "./Profile";
 
 const Layout = () => {
+  const token = localStorage.getItem("token");
+  const householdId = "65e93b29d9f311f834219bed"
```
Comment (nicoleruedger) [inline packages/frontend/src/components/Layout.js:9]: Ideally we find a way to get the householdId dynamically, but you explained the situation - my other comment down below could be worth looking at?

**Review by nicoleruedger — APPROVED** [review]

File: packages/frontend/src/pages/LogIn.js (hunk 1 above)
Comment (AlexiLynn) [reply packages/frontend/src/pages/LogIn.js:33]: You are so right, there is no household id. Unfortunately, even when I add this line it still doesn't work. And yes I made sure all the verbage was houseId (I was curious if having the same name made a difference, apparently not.)
Screenshot 2024-03-08 at 3 50 24 PM

**Comment by swarakurakula** [comment]
looks good alexi! great work
not sure about the householdId part, but we can take a look at that together this weekend!

**Comment by AlexiLynn** [comment]
> looks good alexi! great work not sure about the householdId part, but we can take a look at that together this weekend!

Wonderful. Thank you Swara.

**Review by Sombaudy — APPROVED** [review]
if it works then nice

**Pull request closed without merging**